Assets/MixedRealityToolkit/_Core/Devices/MixedReality/WMRDevice.cs

```diff
@@ -246,12 +271,18 @@ private void UpdateInteractionSource(InteractionSourceState interactionSourceSta
 
 
             //Update Grip
+            //TODO - Review, as we no longer have a MixedRealityCameraParent, this will cause issue.
             if (CameraCache.Main.transform.parent != null)
             {
                 gripPosition = CameraCache.Main.transform.parent.TransformPoint(gripPosition);
                 gripRotation.eulerAngles = CameraCache.Main.transform.parent.TransformDirection(gripRotation.eulerAngles);
             }
             sourceData.Grip.Update(gripPosition, gripRotation);
+
+            // Update Touchpad
+            if (interactionSourceState.touchpadTouched)
```

Comment:
Note: Touchpad (also, Grip, Thumbstick, Menu) are features that may need an API contract / os version check

Reply:
Again, we'll refine once we have an MVP.  The MVP is classes as a single API version with WMR controllers.  Then we can extend to support the next SDK/Controller - OpenVR